--- host-python3-3.7.0/Misc/python-config.sh.in.orig	2018-08-07 10:26:46.120868238 -0400
+++ host-python3-3.7.0/Misc/python-config.sh.in	2018-08-07 10:27:43.408416032 -0400
@@ -33,9 +33,9 @@
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 exec_prefix_real=${prefix_real}
-includedir=$(echo "@includedir@" | sed "s#$prefix#$prefix_real#")
-libdir=$(echo "@libdir@" | sed "s#$prefix#$prefix_real#")
-CFLAGS=$(echo "@CFLAGS@" | sed "s#$prefix#$prefix_real#")
+includedir=$(echo "@includedir@" | sed "s#^$prefix#$prefix_real#")
+libdir=$(echo "@libdir@" | sed "s#^$prefix#$prefix_real#")
+CFLAGS=$(echo "@CFLAGS@" | sed "s#^$prefix#$prefix_real#")
 VERSION="@VERSION@"
 LIBM="@LIBM@"
 LIBC="@LIBC@"
@@ -49,7 +49,7 @@
 PY_ENABLE_SHARED="@PY_ENABLE_SHARED@"
 LDVERSION="@LDVERSION@"
 LIBDEST=${prefix_real}/lib/python${VERSION}
-LIBPL=$(echo "@LIBPL@" | sed "s#$prefix#$prefix_real#")
+LIBPL=$(echo "@LIBPL@" | sed "s#^$prefix#$prefix_real#")
 SO="@EXT_SUFFIX@"
 PYTHONFRAMEWORK="@PYTHONFRAMEWORK@"
 INCDIR="-I$includedir/python${VERSION}${ABIFLAGS}"
