--- python3-3.7.0/setup.py.orig	2018-08-08 08:40:17.829494291 -0400
+++ python3-3.7.0/setup.py	2018-08-08 08:43:30.637714312 -0400
@@ -1627,7 +1627,7 @@
             missing.append('_tkinter')
 
         # Build the _uuid module if possible
-        uuid_incs = find_file("uuid.h", inc_dirs, ["/usr/include/uuid"])
+        uuid_incs = find_file("uuid.h", inc_dirs, [])
         if uuid_incs is not None:
             if self.compiler.find_library_file(lib_dirs, 'uuid'):
                 uuid_libs = ['uuid']
